relay-event-normalization/src/transactions/processor.rs

```diff
@@ -438,13 +334,8 @@ fn end_all_spans(event: &mut Event) -> ProcessingResult {
                 span.timestamp.set_value(event.timestamp.value().cloned());
                 span.status = Annotated::new(SpanStatus::DeadlineExceeded);
             }
-        } else {
-            return Err(ProcessingAction::InvalidTransaction(
-                "spans must be valid in transaction event",
-            ));
```

Comment:
This check is now part of the validation, and all spans at this point are valid.